test/scripts/jenkins_xpack_build_kibana.sh

```diff
@@ -34,7 +34,7 @@ node scripts/functional_tests --assert-none-excluded \
 if [[ -z "$CODE_COVERAGE" ]] ; then
   echo " -> building and extracting default Kibana distributable for use in functional tests"
   cd "$KIBANA_DIR"
-  node scripts/build --debug --no-oss
+  node scripts/build --debug --no-oss --all-platforms
```

Comment:
I'm not clear what this change does.  What platform(s) were we not building that we are now?  And are we testing those other platforms somehow?

Reply:
it's a convenience that will be reverted before merging - you'll see it on the upload report in jenkins